---
[v2] arm64: dts: mt7988: disable eee for aqr 10gphy

Disable EEE for AQR 10G PHYs to prevent Rx error packets from USXGMII.
---
v2: Disable 2.5G/5G EEE modes.
v1: Move openwrt-master files to openwrt-25.12 branch after OpenWrt official
    25.12 branch release.
---

From 9986883644ec72eebab57db8a3fbebb1045ea674 Mon Sep 17 00:00:00 2001
From: Bo-Cun Chen <bc-bocun.chen@mediatek.com>
Date: Fri, 3 Oct 2025 10:39:14 +0800
Subject: [PATCH] arm64: dts: mt7988: disable eee for aqr 10gphy

Without this patch, users will observe the Rx error packet count from
USXGMII RX on the AQR 10G PHY when EEE is enabled.

Signed-off-by: Bo-Cun Chen <bc-bocun.chen@mediatek.com>
---
 arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-aqr.dtso     | 2 ++
 arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-cux3410.dtso | 2 ++
 arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-aqr.dtso     | 2 ++
 arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-cux3410.dtso | 2 ++
 4 files changed, 8 insertions(+)

--- a/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-aqr.dtso
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-aqr.dtso
@@ -26,6 +26,10 @@
 				reset-gpios = <&pio 72 GPIO_ACTIVE_LOW>;
 				reset-assert-us = <100000>;
 				reset-deassert-us = <221000>;
+				eee-broken-10gt;
+				eee-broken-5gt;
+				eee-broken-2_5gt;
+				eee-broken-1000t;
 			};
 		};
 	};
--- a/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-cux3410.dtso
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth1-cux3410.dtso
@@ -27,6 +27,10 @@
 				reset-gpios = <&pio 72 GPIO_ACTIVE_LOW>;
 				reset-assert-us = <100000>;
 				reset-deassert-us = <221000>;
+				eee-broken-10gt;
+				eee-broken-5gt;
+				eee-broken-2_5gt;
+				eee-broken-1000t;
 
 				leds {
 					#address-cells = <1>;
--- a/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-aqr.dtso
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-aqr.dtso
@@ -26,6 +26,10 @@
 				reset-gpios = <&pio 71 GPIO_ACTIVE_LOW>;
 				reset-assert-us = <100000>;
 				reset-deassert-us = <221000>;
+				eee-broken-10gt;
+				eee-broken-5gt;
+				eee-broken-2_5gt;
+				eee-broken-1000t;
 			};
 		};
 	};
--- a/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-cux3410.dtso
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-rfb-eth2-cux3410.dtso
@@ -26,6 +26,10 @@
 				reset-gpios = <&pio 71 GPIO_ACTIVE_LOW>;
 				reset-assert-us = <100000>;
 				reset-deassert-us = <221000>;
+				eee-broken-10gt;
+				eee-broken-5gt;
+				eee-broken-2_5gt;
+				eee-broken-1000t;
 
 				leds {
 					#address-cells = <1>;
